StratisCore.UI/src/app/wallet/send/send.component.ts

```diff
@@ -394,4 +394,30 @@ export class SendComponent implements OnInit, OnDestroy {
   private startSubscriptions() {
     this.getWalletBalance();
   }
+
+  private numberToString(num) {
+    let numStr = String(num);
```

Comment:
we can probably return 0 straign away if passed param is invalid, i.e. `if (isNaN(num)) return '0';`, otherwise this function may return `NaN` which we may not want to pass to the API later on.

Reply:
Do we need a function to do this? `opReturnAmount` is only used in one place, so we could just hardcode it as a string 😃

Reply:
I think Rowan has a point here, lol